core/scripts/vrfv2/testnet/main.go

```diff
@@ -854,6 +855,112 @@ func main() {
 		resp, err := coordinator.GetSubscription(nil, *subID)
 		helpers.PanicErr(err)
 		fmt.Println("sub id", *subID, "balance:", resp.Balance)
+	case "coordinator-withdrawable-tokens":
```

Comment:
I like this approach! I am wondering if it might be better if we add this as part of the `vrf` package in core/services and add some tests to validate it (with a mocked out `isWithdrawable). Perhaps:

```golang
func MaxWithdrawableForOracle(coordinatorAddress, oracleAddress common.Address, isWithdrawable func(amount *big.Int) bool) {
  ...
}
```

This function name isn't great, but I think the approach you've outlined really does return the max amount an oracle can withdraw right?

Reply:
Would it make sense to add a test but keep it in scipts? Or are you thinking something in services would use it eventually?

Yeah this should return the exact amount down to the Juel

Reply:
Ah yeah, I don't think we'd use this in services (unless we automate down the line - in which case it would be easy to port this anyway) but more that I think it would benefit from a unit test. Totally fine if it's in the scripts directory though, I was thinking might be easier to write the test in the `vrf` package.